packer/provision.sh

```diff
@@ -46,10 +44,13 @@ rm /tmp/echidna.tar.gz
 echo "Downloading medusa..."
 sudo apt-get update
 sudo apt-get install -y unzip
```

Comment:
These can probably be removed, unzip is already being installed earlier on
```suggestion
```

Reply:
On a second look, it seems unzip is no longer used in this script. It may be OK to remove from the first install command as well if it's not used anywhere else.